modules/33acrossBidAdapter.js

```diff
@@ -363,16 +410,13 @@ function _getPercentInView(element, topWin, { w, h } = {}) {
  * Viewability contribution to request..
  */
 function ViewabilityContributor(viewabilityAmount) {
-  function contributeViewability(ttxRequest) {
-    const req = Object.assign({}, ttxRequest);
-    const imp = req.imp = req.imp.map(impItem => Object.assign({}, impItem));
-    const banner = imp[0].banner = Object.assign({}, imp[0].banner);
-    const ext = banner.ext = Object.assign({}, banner.ext);
+  function contributeViewability() {
+    const ext = {};
```

Comment:
with this change, the `ext.ttx` is always undefined in following line `const ttx = ext.ttx = Object.assign({}, ext.ttx);`